components/esp_codec_dev/CMakeLists.txt

```diff
@@ -62,6 +62,9 @@ if (CONFIG_CODEC_ZL38063_SUPPORT)
     device/zl38063/example_apps/tw_spi_access.c)
 endif()
 
+# To ignore the compiling warning 'strict-prototypes' while this component is used as an external component in the 'esp components registry'
+set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -Wno-strict-prototypes")
```

Comment:
can we use `target_compile_options` to set the flag for the component? e.g. `target_compile_options(${COMPONENT_LIB} PRIVATE -Wno-strict-prototypes")`